dacapo/examples/distance_task/synthetic_example.py

```diff
@@ -121,76 +123,97 @@
 # NOTE: You may need to delete old config stores if you are re-running this cell with modifications to the configs. The config names are unique and will throw an error if you try to store a config with the same name as an existing config. For the `files` backend, you can delete the `runs_base_dir/configs` directory to remove all stored configs.
 
 # %%
-from dacapo.experiments.datasplits.datasets.arrays import (
-    BinarizeArrayConfig,
-    ZarrArrayConfig,
-    IntensitiesArrayConfig,
-)
-from dacapo.experiments.datasplits import TrainValidateDataSplitConfig
-from dacapo.experiments.datasplits.datasets import RawGTDatasetConfig
 from pathlib import Path
-from dacapo import Options
+from dacapo.experiments.datasplits import DataSplitGenerator
+from funlib.geometry import Coordinate
 
-options = Options.instance()
-runs_base_dir = options.runs_base_dir
+input_resolution = Coordinate(8, 8, 8)
+output_resolution = Coordinate(8, 8, 8)
+datasplit_config = DataSplitGenerator.generate_from_csv(
```

Comment:
Need to add an option have how to do the datasplit if the synthetic data is newly generated (and housed elsewhere).

Reply:
i think it is time to merge and go
and we can code it with them in the flow